caffe2/serialize/file_adapter.cc

```diff
@@ -40,16 +38,17 @@ FileAdapter::RAIIFile::~RAIIFile() {
 
 // FileAdapter directly calls C file API.
 FileAdapter::FileAdapter(const std::string& file_name) : file_(file_name) {
-  const int fseek_ret = fseek(file_.fp_, 0L, SEEK_END);
+  auto fseek_ret = fseek(file_.fp_, 0L, SEEK_END);
```

Comment:
I'm not sure what's the inpact of changing these tbh. Can we keep explicit types?

Reply:
Reverted

Reply:
`auto` is almost always a better choice because it prevents accidentally doing an implicit cast to the wrong type and, as a reviewer, you _know_ the type is correct. I had to look up the API here to understand what was happening.